pkg/envoy/rds_test.go

```diff
@@ -18,12 +19,27 @@ func TestEnvoyOutboundRouteDomainPodChecker(t *testing.T) {
 		getter: createConfigGetterFunc("../../tests/sample-envoy-config-dump-bookbuyer.json"),
 	}
 	pod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "bookstore-v1-74d9cbd7c8-dlmlj",
```

Comment:
Should pod names be consistent with this formatting throughout the tests?

Reply:
I'm not sure it's worth making that a hard rule across the board, but using a pod name like this here would definitely have exposed this bug. If I'm handcrafting a scenario from scratch though I'd have no qualms naming a pod `"a"`. When using Envoy configs pulled from the demo like this, I'm more inclined to name things realistically just so the test scenario is as consistent with itself as possible.

(side note: I'll remove the pod names in these tests since they're not actually needed anymore.)